Makefile

```diff
@@ -38,6 +45,10 @@ REGISTRY ?= quay.io/stolostron
 TAG ?= latest
 IMAGE_NAME_AND_VERSION ?= $(REGISTRY)/$(IMG)
 
+# Github host to use for checking the source tree;
+# Override this variable ue with your own value if you're working on forked repo.
+GIT_HOST ?= github.com/stolostron
```

Comment:
What needs this? Just out of curiosity

Reply:
It's use in `e2e-build-instrumented`, but honestly it's probably not needed--I was more following patterns in other repos and decided to stick it in there to make it match. 🙂

Reply:
That's a good reason :D